src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -113,12 +114,27 @@ private string BuildModuleKey(string projectGuid)
             await SynchronizeSuppressedIssues();
         }
 
+        private Task DoInitialFetch()
+        {
+            // We might not have connected to the server at this point so if necessary
+            // wait before trying to fetch the issues
+            int retryCount = 0;
+            while (!this.sonarQubeService.IsConnected && retryCount < 30)
```

Comment:
Could this code be needed somewhere else? Isn't it a good idea to put it inside the SonarQubeService as something like:
`public Task WaitForConnection()`

Reply:
We might need it elsewhere, but I'd rather keep it local for the time being. This is a quick and ugly workaround to unblock the release. We could probably come up with a more elegant solution for the general purpose case.